pkg/kubelet/apis/config/types.go

```diff
@@ -108,6 +108,9 @@ type KubeletConfiguration struct {
 	// readOnlyPort is the read-only port for the Kubelet to serve on with
 	// no authentication/authorization (set to 0 to disable)
 	ReadOnlyPort int32
+	// readOnlyBindAddress is the address for the Kubelet to serve on with
+	// no authentication/authorization (set to 0.0.0.0 to serve on all interfaces)
```

Comment:
text has no information than one in command line =). IPv6 address example would be useful.